tests/ChannelClientTests.cs

```diff
@@ -812,5 +818,67 @@ public async Task TestUnarchiveChannelForMemberAsync()
 
             archivedChannels.Channels.Should().NotContain(c => c.Channel.Cid == channel.Cid);
         }
+
+        [Test]
+        public async Task TestQueryChannelsWithoutMembersLimit()
+        {
+            var queryChannelResponse = await _channelClient.QueryChannelsAsync(new QueryChannelsOptions
+            {
+                Filter = new Dictionary<string, object>()
+                {
+                    { "cid", _channel2.Cid },
+                },
+                UserId = _user1.Id,
+            });
+            queryChannelResponse.Channels.Should().HaveCount(1);
+            queryChannelResponse.Channels.First().Members.Should().HaveCount(3);
+        }
+
+        [Test]
+        public async Task TestQueryChannelsWithMembersLimit()
+        {
+            var queryChannelResponse = await _channelClient.QueryChannelsAsync(new QueryChannelsOptions
+            {
+                Filter = new Dictionary<string, object>()
+                {
+                    { "cid", _channel2.Cid },
+                },
+                MemberLimit = 1,
+                UserId = _user1.Id,
+            });
+            queryChannelResponse.Channels.Should().HaveCount(1);
+            queryChannelResponse.Channels.First().Members.Should().HaveCount(1);
+        }
+
+        [Test]
+        public async Task TestQueryChannelsWithoutMessageLimit()
+        {
+            var queryChannelResponse = await _channelClient.QueryChannelsAsync(new QueryChannelsOptions
+            {
+                Filter = new Dictionary<string, object>()
+                {
+                    { "cid", _channel2.Cid },
+                },
+                UserId = _user1.Id,
+            });
+            queryChannelResponse.Channels.Should().HaveCount(1);
+            queryChannelResponse.Channels.First().Messages.Should().HaveCount(5);
+        }
+
+        [Test]
+        public async Task TestQueryChannelsWithMessageLimit()
```

Comment:
I forgot the Async postfix for the test methods. I'll add them first thing in the morning